### ISSUE TITLE
Event Entry

### ISSUE THREAD

**Issue by jlindenthaler** [GoldenCheetah/GoldenCheetah#4192]
Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

Hey team,

I have only recently returned to GC and am building my plan for the year and tried to enter an event and the calendar will not let me go beyond Jan 2016.

Josh

**Comment by amtriathlon** [comment]
Please, use https://groups.google.com/forum/#!forum/golden-cheetah-users to ask questions or get help.
Thanks.
